tools/configure_server_settings.rb

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+require File.expand_path("../config/environment", __dir__)
+require 'trollop'
+
+opts = Trollop.options(ARGV) do
+  banner "USAGE:   #{__FILE__} -s <server id> -p <settings path separated by a /> -v <new value>\n" \
+         "Example: #{__FILE__} -d -s 1 -p reporting/history/keep_reports -v 42"
+
+  opt :dry_run,  "Dry Run",                            :short => "d"
+  opt :serverid, "Server Id",                          :short => "s", :default => 0
```

Comment:
Nitpicky but we call it server_id in the prior script in the `USAGE` line, we should do that here too.